server/routerlicious/packages/protocol-base/src/quorum.ts

```diff
@@ -220,7 +219,9 @@ export class QuorumProposals extends TypedEventEmitter<IQuorumProposalsEvents> i
     }
 
     /**
-     * Proposes a new value. Returns a promise that will resolve when the proposal is either accepted or rejected.
+     * Proposes a new value. Returns a promise that will either:
+     * - Resolve when the proposal is accepted
+     * - Reject if the proposal fails to send or if the QuorumProposals is disposed
```

Comment:
Is there explicit "reject message" that Readme talks about? I see container handling the proposal and throwing error, but not explicit message.

It would be nice to update Readme and maybe capture github issue conversation on assumptions and constraints. Either I am misinterpreting the language there or it's out of date.

Reply:
Good catch, they are stale -- I removed them in #9092.  Cleaned up in latest.